.eleventy.js

```diff
@@ -80,6 +80,15 @@ module.exports = function(config) {
     return content
   })
 
+  // After executing {% include "practices.njk"%}, the paths in the <iframe> with the demo are broken, because in the content these paths are specified relatively, and the {% include %} function takes the practices to a higher level. This transformation is necessary to edit the paths.
```

Comment:
```suggestion
  // Правит пути к демкам внутри советов
```
Кажется можно проще…